packages/types/surveys.ts

```diff
@@ -679,6 +679,9 @@ export type TSurveyQuestionSummaryNps = z.infer<typeof ZSurveyQuestionSummaryNps
 export const ZSurveyQuestionSummaryCta = z.object({
   type: z.literal("cta"),
   question: ZSurveyCTAQuestion,
+  impressions: z.number(),
```

Comment:
since we normally suffix numbers with `Count`, I think this also makes sense here.
So I would suggest doing these renamings:
`impressions` -> `impressionCount`
`clicks` -> `clickCount`
`skips` -> `skipCount`